packages/vscode-extension/src/project/DeviceSessionsManager.ts

```diff
@@ -16,6 +17,7 @@ import { disposeAll } from "../utilities/disposables";
 import { DeviceId, DeviceSessionsManagerState } from "../common/Project";
 
 const LAST_SELECTED_DEVICE_KEY = "last_selected_device";
+const SWITCH_DEVICE_THROTTLE_MS = 300;
```

Comment:
```suggestion
const SWITCH_DEVICE_THROTTLE_MS = 300;

```
```suggestion
const SWITCH_DEVICE_THROTTLE_MS = 500;
```

after incriseing the timeout I'm unable to cause the problem using the shortcut